### DISCUSSION TITLE
[Bug]: Battery Backup power SOC

### DISCUSSION THREAD

**Discussion by scorpio2k2** [wlcrs/huawei_solar#629] [General]
### Describe the issue

Hello,

I have the following settings for battery and in the morning, when the inverted woke up, it started to change the battery from grid.
DO you have any ideas?  I don't use TOU because the electricity price is the same the whole day. 

![Screenshot 2024-01-29 at 13 15 56](https://github.com/wlcrs/huawei_solar/assets/915081/9fd6e4d9-1a6c-4c53-91da-5b07705938d3)

### Bescribe your Huawei Solar Setup

Inverter Type: SUN2000-4KTL-L1
Inverter Firmware version: V200R001C00SPC130
SDongle present: yes 
Power meter present: single phase 
Battery: LUNA2000 10kWh
Battery Firmware version: V100R002C00SPC119

### How do you connect to the inverter?

Via the SDongle, wireless connection

### Upload your Diagnostics File

[config_entry-huawei_solar-8a04cf4c1fff0c72c161bb5c7d4d10e5.json.txt](https://github.com/wlcrs/huawei_solar/files/14086914/config_entry-huawei_solar-8a04cf4c1fff0c72c161bb5c7d4d10e5.json.txt)

### Upload your relevant debug logs

```text
waiting to happen.
```

### Please confirm the following:

- [X] I'm running the latest release of Home Assistant.
- [X] I'm running the latest release of this integration.
- [X] I did not find an existing issue describing this problem.
- [X] I did upload the diagnostics-file that I could retrieve from  the 'Devices & Services Page'
- [ ] I did upload the relevant debug logs (via 'Enable Debug Logging'-feature or by manually configuring HA logging)

**Comment by philkunz** [comment]
I have the same issue I think.
